minigalaxy/game.py

```diff
@@ -29,45 +29,36 @@ def get_install_directory_name(self):
     def load_minigalaxy_info_json(self):
         json_dict = {}
         if os.path.isfile(self.status_file_path):
-            staus_file = open(self.status_file_path, 'r')
-            json_dict = json.load(staus_file)
-            staus_file.close()
+            with open(self.status_file_path, 'r') as status_file:
+                json_dict = json.load(status_file)
         return json_dict
 
     def save_minigalaxy_info_json(self, json_dict):
         if not os.path.exists(CONFIG_GAMES_DIR):
             os.makedirs(CONFIG_GAMES_DIR, mode=0o755)
-        status_file = open(self.status_file_path, 'w')
-        json.dump(json_dict, status_file)
-        status_file.close()
+        with open(self.status_file_path, 'w') as status_file:
+            json.dump(json_dict, status_file)
 
     @staticmethod
     def __strip_string(string):
         return re.sub('[^A-Za-z0-9]+', '', string)
 
     def is_installed(self, dlc_title="") -> bool:
-        installed = False
         if dlc_title:
             dlc_version = self.get_dlc_info("version", dlc_title)
-            installed = True if dlc_version else False
+            return bool(dlc_version)
```

Comment:
Here we introduced second return statement to the function which previously had only one return. I prefer functions to have only one return. That way if future modifications are introduced we will make sure that they are executed in every function run. I know that we already have many functions with multiple returns, but I think that we should work on reducing that number.

Reply:
I'm not sure I understand the reasoning but I'll revert these changes if that's how it's done in this project